vue/components/ui/molecules/calendar/calendar.vue

```diff
@@ -0,0 +1,296 @@
+                            class="cell"
+                            v-bind:class="circleClass(week[d - 1].day, week[d - 1].month)"
+                            v-for="d in 7"
+                            v-bind:key="d - 1"
+                            v-on:click="onCellClick(week[d - 1])"
+                        >
+                            <div class="circle">
+                                {{ week[d - 1].day }}
+                            </div>
+                        </td>
+                    </tr>
+                </tbody>
+            </table>
+        </div>
+    </div>
+</template>
+
+<style lang="scss" scoped>
+@import "css/variables.scss";
+
+.calendar {
+    border: 1px solid #e4e8f0;
+    border-radius: 6px 6px 6px 6px;
+    box-shadow: 0px 0px 30px rgba(21, 21, 21, 0.1);
+    display: flex;
+    flex-direction: column;
+    position: absolute;
+    transition: opacity 0.35s ease-in-out;
+    width: 350px;
+}
+
+.calendar > .calendar-header {
+    align-items: center;
+    display: flex;
+    justify-content: space-between;
+    padding: 10px 10px 10px 10px;
+}
+
+.calendar > .calendar-header > .header-center {
+    align-items: center;
+    display: inline-block;
+    display: flex;
+}
+
+.calendar > .calendar-content {
+    display: flex;
+    font-weight: 400;
+    height: 100%;
+    padding: 0px 20px 10px 20px;
+}
+
+.calendar > .calendar-content .calendar-table {
+    width: 100%;
+}
+
+.calendar > .calendar-content .calendar-table .table-header .row {
+    height: 40px;
+}
+
+.calendar > .calendar-content .calendar-table .table-header .row .heading {
+    font-weight: 400;
+}
+
+.calendar > .calendar-content .calendar-table .table-body .row .cell {
+    height: 40px;
+    padding: 0px;
+    text-align: center;
+    width: 40px;
+}
+
+.calendar > .calendar-content .calendar-table .table-body .row .cell.clickable {
+    cursor: pointer;
+}
+
+.calendar > .calendar-content .calendar-table .table-body .row .cell .circle {
+    border: 3px solid transparent;
+    border-radius: 50%;
+    box-sizing: border-box;
+    color: #1d2631;
+    height: 40px;
+    line-height: 35px;
+    margin: auto;
+    user-select: none;
+    width: 40px;
+}
+
+.calendar > .calendar-content .calendar-table .table-body .row .cell.clickable:not(.selected):hover .circle {
+    background-color: #ecf0f3;
+}
+
+.calendar > .calendar-content .calendar-table .table-body .row .cell:not(.clickable) .circle {
+    color: #bac2cb;
+}
+
+.calendar > .calendar-content .calendar-table .table-body .row .cell.selected .circle {
+    background-color: #1d2631;
+    border: 3px solid #ecf0f3;
+    color: #ffffff;
+}
+
+.calendar .select ::v-deep .select-container .select-button {
+    background-color: #ffffff;
+}
+</style>
+
+<script>
+export const Calendar = {
+    props: {
+        /**
+         * The value of input date in String or Date format.
+         * Example: "2020/12/31"
+         */
+        value: {
+            type: Date | String,

```

Comment:
Don't use `Date` because it's a mutable object. This means it's prone to this kind of error:
- a = Date(10)
- v-bind:stuff="a"
- a.setMonth(3)
- v-bind doesn't trigger